nautobot/core/views/generic.py

```diff
@@ -157,6 +157,7 @@ def get_filter_params(self, request):
     def get_required_permission(self):
         return get_permission_for_model(self.queryset.model, "view")
 
+    # TODO: remove this as well?
```

Comment:
Maybe we need/want a NautobotYAMLRenderer too? Thought that might necessitate a hard fork from the device-types library repo.

Reply:
I'm thinking yes. For what it's worth there's `djangorestframework-yaml` from the same author (whose work you didn't end up using), of note specifically is the encoder class: https://github.com/jpadilla/django-rest-framework-yaml/blob/master/rest_framework_yaml/encoders.py

Reply:
That looks like a larger body of work, so I've opened #3772.